monitoring/types.go

```diff
@@ -14,6 +14,7 @@ const (
 	subsystemDisk            = "disk"
 	subsystemInstanceManager = "instance_manager"
 	subsystemManager         = "manager"
+	subsystemBackups         = "backups"
```

Comment:
Suggest `subsystemBackup` instead

Reply:
Fix in d156d1fdaa05700e384ff7ab0ea429f15ee27ab4